docs/bulkloading.txt

```diff
@@ -55,44 +49,36 @@ When id blocks are frequently allocated by many JanusGraph instances in parallel
 
 *Rule of thumb*: Set this value to be as large feasible to not have to wait too long for unrecoverable failures. The only downside of increasing it is that JanusGraph will try for a long time on an unavailable storage backend cluster.
 
-Optimizing Writes and Reads
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+==== Optimizing Writes and Reads
 
-Buffer Size
-+++++++++++
+===== Buffer Size
 
 JanusGraph buffers writes and executes them in small batches to reduce the number of requests against the storage backend. The size of these batches is controlled by `storage.buffer-size`. When executing a lot of writes in a short period of time, it is possible that the storage backend can become overloaded with write requests. In that case, increasing `storage.buffer-size` can avoid failure by increasing the number of writes per request and thereby lowering the number of requests. 
 
 However, increasing the buffer size increases the latency of the write request and its likelihood of failure. Hence, it is not advisable to increase this setting for transactional loads and one should carefully experiment with this setting during bulk loading.
 
-Read and Write Robustness
-+++++++++++++++++++++++++
+===== Read and Write Robustness
 
 During bulk loading, the load on the cluster typically increases making it more likely for read and write operations to fail (in particular if the buffer size is increased as described above). 
 `storage.read-attempts` and `storage.write-attempts` configure how many times JanusGraph will attempt to execute a read or write operation against the storage backend before giving up. If it is expected that there is a high load on the backend during bulk loading, it is generally advisable to increase these configuration options.
 
 `storage.attempt-wait` specifies the number of milliseconds that JanusGraph will wait before re-attempting a failed backend operation. A higher value can ensure that operation re-tries do not further increase the load on the backend.
 
-//Tools
-//~~~~~
+//=== Tools
 
-//JanusGraph-Hadoop
-//^^^^^^^^^^^^
+//==== JanusGraph-Hadoop
 
 //For very large graphs the best option to load data efficiently is <<hadoop, JanusGraph-Hadoop>> using one of the supported input format and specifying JanusGraph as the output format.
 
-//BatchGraph
-//^^^^^^^^^^
+//==== BatchGraph
 
 //For medium size graph datasets (up to 100s million edges), Tinkerpop ' http://tinkerpop.apache.org/docs/$MAVEN{tinkerpop.version}/#_batchgraph[BatchGraph] is a useful tool for bulk loading data into JanusGraph from a single machine through JanusGraph's native Blueprints interface. BatchGraph effectively caches externally provided vertex ids to eliminate reads against JanusGraph. This allows bulk loading with minimal read load.
```

Comment:
Batchgraph is TP2.0 concept. Now i think we have BulkVertexLoaderProgram or something like that. Lets fix this discussion or create an issue to fix later.